packages/firestore/src/lite/database.ts

```diff
@@ -59,14 +59,17 @@ declare module '@firebase/component' {
  *
  * Do not call this constructor directly. Instead, use {@link getFirestore}.
  */
-export class FirebaseFirestore implements FirestoreService {
+export class Firestore implements FirestoreService {
+  /**
+   * whether it's a firestore or firestore-lite instance
```

Comment:
@egilmorez Does it look good to you?

Reply:
If it's public, we should cap "Whether" and either:

-- backtick the types of instances as literals
-- just use the capitalized product names.

Reply:
They share the same type name, so I changed to use the capitalized product names.